chart/templates/registry-facade-configmap.yaml

```diff
@@ -35,10 +35,6 @@ data:
             "store": "/mnt/cache/registry",
             "requireAuth": false,
             "staticLayer": [
-                {
```

Comment:
@csweichel: Would it be better if we omit this change for now? To be backwards compatible, we need to deploy the new version of `server` before we can remove this configuration from the registry facade, right? If we would deploy this config change before `server` has been updated then we wouldn't have a `supervisor` layer at all.

Thus, it is probably better we keep this as a deprecated state and remove this in 2 weeks or so. WDYT?

Reply:
Good catch. That is indeed correct - let's remove this by the end of the month.